### DISCUSSION TITLE
Towny PvP

### DISCUSSION THREAD

**Discussion by OfficialBasic** [TownyAdvanced/Towny#6053] [Q&A]
Recently moved server and towny kinda bugged, and i kinda need help before i go nuts.

![towny](https://user-images.githubusercontent.com/108149806/180326264-33bb1c27-66fb-44c7-9b1a-ed7d83dea784.png)

**Comment by LlmDl** [comment]
Maybe you changed your world name?

**Reply by OfficialBasic** [reply]
No not really

**Reply by LlmDl** [reply]
So what's the screenshot about?
